bindata/network/openshift-sdn/sdn-public.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ConfigMap
+apiVersion: v1
+immutable: true
```

Comment:
Everything in this configmap will be generated once and left alone for the life time of the cluster? With this feature, you cannot edit the configmap. What happens if a CNI gets updated with a new feature that contradicts the existing configmap and we have to update this configmap? You can delete it and recreate it I guess.